index.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+let correctAnswerCount = 0;                                                                             //正解解答数格納用の変数宣言
```

Comment:
グローバルな変数はここにすべて定義してしまいましょう。

```suggestion
const quizzes = [];
let correctAnswerCount = 0;                                                                             //正解解答数格納用の変数宣言
let count = 0;                                                                                          //取得クイズカウント用変数宣言
let questionNumber = 1;
```